linux_os/guide/services/obsolete/r_services/no_user_host_based_files/ansible/shared.yml

```diff
@@ -3,26 +3,28 @@
 # strategy = restrict
 # complexity = low
 # disruption = low
-- name: "Find local mount points"
-  shell: |
-      set -o pipefail
-      df --local | awk '{print $6}' | grep -v Mounted | grep -v '^/dev' || true
-  register: local_mount_points
 
-- name: "Detect the .shosts files on the system"
-  find:
-      paths: "{{ item }}"
-      recurse: yes
-      patterns: [".shosts"]
-      hidden: yes
-      file_type: "file"
-  check_mode: no
-  with_items: "{{ local_mount_points.stdout_lines }}"
-  register: shosts_locations
+{{{ ansible_create_list_of_local_paths(list_name="search_paths") }}}
 
-- name: "Remove .shosts Files"
-  file:
-      path: "{{ item.path }}"
-      state: absent
-  with_items: "{{ shosts_locations.results | map(attribute='files') | list }}"
-  when: shosts_locations is success
+- name: "{{{ rule_title }}} - Define Rule Specific Facts"
+  ansible.builtin.set_fact:
+    user_shosts_files:
+      - /.shosts
+
+- name: "{{{ rule_title }}} - Find All .shosts Files in Local File Systems"
+  ansible.builtin.command:
+    cmd: find {{ item }} -xdev -type f -name ".shosts"
```

Comment:
Will this be reported as not permitted shell command?

Reply:
No. It is using the `cmd` module and the test only cares about `shell` module, so it won't be reported as an issue.